container/network.go

```diff
@@ -0,0 +1,32 @@
+package container
+
+import (
+	"github.com/nspcc-dev/neofs-sdk-go/netmap"
+)
+
+// ApplyNetworkConfig applies network configuration to the
+// container. Changes the container if it does not satisfy
+// network configuration.
+//
+// Network config must contain unique parameters. Duplicated
+// network parameters lead to undefined behaviour.
+func (x *Container) ApplyNetworkConfig(cfg netmap.NetworkInfo) {
```

Comment:
Lets make it a function. Same for assert.

Reply:
why? it now uses private fields